src/__tests__/radium-test.js

```diff
@@ -276,6 +276,36 @@ describe('Radium blackbox tests', () => {
     expect(nav.style.color).to.equal('blue');
   });
 
+  it.only('passes snapshot to the componentDidUpdate of the component, Issue #985', () => {
+    const SNAPSHOT = 'SNAPSHOT';
+
+    class SnapshotComp extends Component {
+      componentDidMount() {
+        this.forceUpdate();
+      }
+
+      componentDidUpdate(props, state, snapshot) {
+        expect(snapshot).to.equal(SNAPSHOT);
+      }
+
+      getSnapshotBeforeUpdate() {
+        return SNAPSHOT;
+      }
+
+      render() {
+        return null;
+      }
+    }
+
+    sinon.spy(SnapshotComp.prototype, 'componentDidUpdate');
+
+    const TestComponent = Radium(SnapshotComp);
+
+    TestUtils.renderIntoDocument(<TestComponent />);
+
+    assert(SnapshotComp.prototype.componentDidUpdate.calledOnce);
```

Comment:
Let's remove `assert` and replace with `sinon-chai` asserts which we use everywhere else.

So, something like:

```js
expect(SnapshotComp.prototype.componentDidUpdate).to.have.been.calledOnce;
// OR
expect(SnapshotComp.prototype.componentDidUpdate).to.have.callCount(1);
```

Reply:
done.